src/router/router.js

```diff
@@ -68,7 +68,11 @@ const allRoutes = [
     path: '/event/:eventId',
     name: 'single-event',
     component: SingleEventView,
-    props: true,
+    props(route) {
```

Comment:
why is this needed?

Reply:
Without this, the event ID is passed to components not as a number but as a string. That was causing issues where we had to use parseInt() and have eslint ignore that line. This just makes it so we don't have to parseInt.